tests/unit/test_task_remote_mgr.py

```diff
@@ -23,10 +23,37 @@
 from cylc.flow.network.client_factory import CommsMeth
 from cylc.flow.task_remote_mgr import (
     REMOTE_FILE_INSTALL_DONE, REMOTE_INIT_IN_PROGRESS, TaskRemoteMgr)
+from cylc.flow.workflow_files import WorkflowFiles, get_workflow_srv_dir
 
 Fixture = Any
 
 
+@pytest.mark.parametrize(
+    'comms_meth, expected',
+    [
+        (CommsMeth.SSH, True),
+        (CommsMeth.ZMQ, True),
+        (CommsMeth.POLL, False)
+    ]
+)
+def test__remote_init_items(comms_meth: CommsMeth, expected: bool):
+    """Test _remote_init_items().
+
+    Should only includes files under .service/
+    """
+    reg = 'barclay'
```

Comment:
FYI: `reg` is Cylc 7 slang resulting from the `cylc reg[ister]` command, `id` is Cylc 8.

Reply:
I know but couldn't resist the opportunity to slip in a Star Trek character's name (Lt Reg Barclay)